packages/lib/gpcircuits/src/proto-pod-gpc.ts

```diff
@@ -260,6 +264,16 @@ export function arrayToProtoPODGPCCircuitParam(
   );
 }
 
+/**
+ * Mapping computing the maximum number of virtual entries from given GPC
+ * parameters.
+ */
+export function paramMaxVirtualEntries<
```

Comment:
Is there value to this being calculated externally with this function, and passed into functions like outputsFromPublicSignals below?  Vs. it being calculated only internally in this layer?

Reply:
I think it will come in handy in the GPC layer.